From 653c39b0b30b2a329db5041e558cfc97c03d6bfb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 17 Feb 2021 19:18:16 -0800
Subject: [PATCH] do not build examples

examples need ucontext and some architectures on musl do not have it
implemented e.g. riscv

Upstream-Status: Inappropriate [Arch-Specific]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5d9c4dc..53e1615 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,6 @@ default: all
 all:
 	@$(MAKE) -C src
 	@$(MAKE) -C test
-	@$(MAKE) -C examples
 
 .PHONY: all install default clean test
 .PHONY: FORCE cscope
-- 
2.31.1

